### PR TITLE
Singlevideopage features

### PR DESCRIPTION
**Pull request by prankurpandeyy**
Things to watch :

- Can add video to likes and watchlater (can't see in page but response checked in console is positive)
- Modal for making playlists can enter playlist name and make a playlists for the same name with a checkbox (functinality witll extend later)
- code quality and folder structure

### CODE REVIEW

File: slate-video-library/src/Components/Context/SingleVideopageContext.js [hunk 1]
```diff
@@ -47,18 +70,68 @@ function SingleVideopageContext({ children }) {
...
+
+  const likedVideo = async (videoData, dispatch) => {
+    const response = await axios({
+      method: "POST",
+      url: `/api/user/likes`,
+      headers: { authorization: localStorage.getItem("token") },
+      data: { video: videoData },
+    });
+
+    dispatch({ type: "SINGLEVIDEOLIKES", payload: response.data.likes });
```
Comment (Adarsh-Patel-Dev) [inline slate-video-library/src/Components/Context/SingleVideopageContext.js:83]: use underscore or hyphens to join two words for readability like `SINGLE_VIDEO_LIKES`

File: slate-video-library/src/Components/PlaylistModal/PlaylistModal.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          onChange={(e) =>
+            dispatch({ type: "INPUTSTATE", payload: e.target.value })
+          }
+        />{" "}
+        <span class="material-icons icons" onClick={postPlaylist}>
+          playlist_add
+        </span>
+      </div>
+
+      {playlists.map((inputtext) => (
```
Comment (Adarsh-Patel-Dev) [inline slate-video-library/src/Components/PlaylistModal/PlaylistModal.js:37]: use camelCase convention for naming

**Review by Adarsh-Patel-Dev — APPROVED** [review]
code structure and code quality is pretty good
You can merge after suggested minor changes

File: slate-video-library/src/Components/Context/SingleVideopageContext.js [hunk 3]
```diff
@@ -11,33 +11,56 @@ const singleVideoContext = createContext();
 export const useSingleVideoContext = () => useContext(singleVideoContext);
 
 function SingleVideopageContext({ children }) {
+  const token = `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiJlMzIzZmY2MC1hMTUzLTQ0MTYtYmEyNS0zNDQ0ZGI1NjliOWMiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ._-fah2UEuueLmRHHl5uV4CYhiQdODX6neUkGbfTvtFM`;
```
Comment (nikhiljugale007) [inline slate-video-library/src/Components/Context/SingleVideopageContext.js:14]: Suggestions: remove token of you are not using or put it in env file

**Review by nikhiljugale007 — APPROVED** [review]
Functionality working properly in ui.
Code is also good. Some minor suggestions.

**Pull request merged**